Python/samples/cartpole/main.py

```diff
@@ -387,6 +388,18 @@ def CreateSession(
                 episode += 1
             elif event.type == "EpisodeStep":
                 iteration += 1
+                if sim_speed > 0:
+                    if sim_speed_variance > 0:
+                        delay = np.random.uniform(
+                            low = np.max([0,sim_speed - sim_speed_variance]),
+                            high = sim_speed + sim_speed_variance
+                            )
+                        print('stochastic delay: {}s'.format(delay))
+                        time.sleep(delay)
+                    else:
+                        delay = sim_speed
+                        print('delay: {}s'.format(delay))
```

Comment:
Would be useful to log the sim-delay as well? Might help with debugging purposes.

Reply:
you mean adding delay to the logs when user does --log_iterations?

Reply:
yeah, so that users can keep track of how much delay was added for that iteration.

Reply:
Thank you, sim_speed_delay is now logged